moz_preprocess/bq_utils.py

```diff
@@ -2,7 +2,7 @@
 from google.cloud import bigquery
 
 
-path_to_parent_directory = os.path.dirname(os.path.realpath(__file__))
+path_to_parent_directory = os.path.dirname(os.path.realpath(os.getcwd()))
```

Comment:
When I tried it last week, I got `NameError: name '__file__' is not defined`. But when I changed it back from `os.getcwd()` to `__file__` just now to double check, it did not throw an error.(weird) I still lean towards keeping `os.getcwd()` in case someone can get the error. What do you think?

Reply:
The only reason I'd be hesitant to change this is that I would expect `__file__` to always point to the location of the file, but I thought that `os.getcwd()` was expected to get the current directory where Python was running. So you could get into a situation where `os.getcwd()` doesn't actually return the path to the file.

Reply:
Okay. I see what you mean. I think I got the file path with `os.getcwd()` because the current directory happened to match the file path. I am reverting it back to `__file__`.